graspologic/nominate/VNviaSGM.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+from ..match import GraphMatch as GMP
+from sklearn.base import BaseEstimator
+
+
```

Comment:
Pedigo still wants both our classes to inherit from the same base for consistency with other modules, even if the base class isn't providing any real functionality. So before merge I think you'll have somehow add the baseVN class and inherit from it. Shouldn't change code behavior at all.

Reply:
Sounds good, are you done modifying the Base class you wrote? If so I can just copy it

Reply:
Yeah, its still pretty basic. You can modify it however you want without effecting me as long as the methods already there stay and no abstract methods are added

Reply:
I chose to leave this in for now because i wanted to get @asaadeldin11 opinion on it. I think our modules are different enough that having the base estimator in common is enough